SlevomatCodingStandard/Sniffs/Classes/SingleLineMethodDeclarationWhenPossibleSniff.php

```diff
@@ -0,0 +1,104 @@
+<?php declare(strict_types = 1);
+
+namespace SlevomatCodingStandard\Sniffs\Classes;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use SlevomatCodingStandard\Helpers\FunctionHelper;
+use SlevomatCodingStandard\Helpers\TokenHelper;
+use function assert;
+use function in_array;
+use function is_bool;
+use function is_int;
+use function is_string;
+use function preg_replace;
+use function rtrim;
+use function sprintf;
+use function str_replace;
+use function strlen;
+use const T_FUNCTION;
+use const T_OPEN_CURLY_BRACKET;
+use const T_SEMICOLON;
+use const T_WHITESPACE;
+
+class SingleLineMethodDeclarationWhenPossibleSniff implements Sniff
```

Comment:
What about RequireSingleLineMethodDeclarationSniff? I don't like this name :)

Reply:
Sure, makes sense :) I didn't really think about it when copying it over here :D